### DISCUSSION TITLE
Is there a way to implement a custom copy button ?

### DISCUSSION THREAD

**Discussion by alexc957** [code-hike/codehike#388] [Q&A] [answered]
Hello Community!
I am trying to implement a custom button for code blocks. I am using `Code Hike` with `next-mdx-remote`. 
I saw some examples using css to add styling to code hike components, but I was wondering  if there is a way to implement a custom button since I need to show something after the user clicks the button and a tooltip when it hovers over it.

**Comment by pomber** [comment, marked as answer]
Hi! Only customizations that can be done with CSS, for example you could try to make the tooltip with an `:after` pseudo-selector. That said, we can consider changing the HTML of the button to make it easier to customize (with CSS).

**Reply by alexc957** [reply]
Hi @pomber , thanks for the answer.
